templates/web-application/.storybook/main.ts

```diff
@@ -0,0 +1,55 @@
+import type { StorybookConfig } from "@storybook/react-webpack5";
+import type  { Configuration } from "webpack";
+
+const config: StorybookConfig = {
+  stories: [
+    "../src/**/*.mdx",
+    "../src/**/*.stories.@(js|jsx|ts|tsx)"
+  ],
+  addons: [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions",
```

Comment:
Do we want this plugin by default? According to our testing, we don't wan to use SB interactions tests at the moment.

Our stack is:
- Jest + JSDom for interaction tests
- Chromatic for visual regression tests